setup.py

```diff
@@ -8,20 +8,31 @@
 from setuptools import setup, find_packages
 
 
-def get_version():
+# Make sources available using relative paths from this file's directory.
+#  dirname can return '', so use 'or' below to avoid chdir on empty
+#  while being succinct.
+os.chdir(os.path.dirname(__file__) or os.getcwd())
```

Comment:
`os.path.dirname(os.path.abspath(__file__))` is the way to circumvent this issue I know of.